paper.bib

```diff
@@ -0,0 +1,53 @@
+@misc{frontier,
+    author = {Barb Helland},
+    title = {At the Frontier: DOE Supercomputing Launches the Exascale Era},
+    year = {2022},
+    publisher = {United States Department of Energy Office of Science},
+    journal = {Website},
+    url = {https://www.energy.gov/science/articles/frontier-doe-supercomputing-launches-exascale-era}
+}
+
+@misc{official_solution,
+    author = {tjwrona1992},
+    title = {What is the proper way to document a cmake module},
+    year = {2019},
+    publisher = {StackOverflow},
+    journal = {Stackk Overflow Forum},
+    url = {https://stackoverflow.com/a/54671996}
+}
+
+@misc{sphinx_plugin,
+    author = {Kitware},
+    title = {Sphinx Domain for Modern CMake},
+    year = {2022},
+    publisher = {GitHub},
+    journal = {GitHub Repository},
+    url = {https://github.com/scikit-build/moderncmakedomain}
```

Comment:
We should probably look into integrating with this domain in the future as an optional extension since it seems a lot of "official" CMake documentation uses it